packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/gateway/L1WethGateway.sol

```diff
@@ -45,7 +45,8 @@ contract L1WethGateway is L1ArbitrumExtendedGateway {
         l2Weth = _l2Weth;
     }
 
-    function createOutboundTx(
+    function createOutboundTxCustomRefund(
+        address _refundTo,
```

Comment:
we added the `refundTo` option but don't use it in `sendTxToL2(`

Reply:
also seems like we don't expose a `function outboundTransferCustomRefund(` here

Reply:
> we added the `refundTo` option but don't use it in `sendTxToL2(`

will fix

> also seems like we don't expose a `function outboundTransferCustomRefund(` here like we do in the others, any reason for that?

L1WethGateway doesn't override `outboundTransfer` currently too

Reply:
yea, we don't override that one since we don't need the reentrancy guard for that one (we know the weth9 code doesn't have a reentrancy)